prow/cmd/peribolos/main.go

```diff
@@ -82,6 +86,11 @@ func (o *options) parseArgs(flags *flag.FlagSet, args []string) error {
 	flags.IntVar(&o.tokensPerHour, "tokens", defaultTokens, "Throttle hourly token consumption (0 to disable)")
 	flags.IntVar(&o.tokenBurst, "token-burst", defaultBurst, "Allow consuming a subset of hourly tokens in a short burst")
 	flags.StringVar(&o.dump, "dump", "", "Output current config of this org if set")
```

Comment:
Should all these new flags be incompatible with dump? It seems odd that you'd want to take action, while also dumping a config of a specified org.

Reply:
Technically yes (and many of the older ones too)